.github/workflows/ci-release-gate.yml

```
@@ -38,180 +38,216 @@ jobs:
-        uses: ./.github/actions/sbom-and-attest
-        with:
-          image_name: docker.io/${{ vars.DOCKER_HUB_USERNAME }}/${{ matrix.container_name }}
-          image_digest: ${{ matrix.digest }}
-          registry: docker.io
-          registry_username: ${{ vars.DOCKER_HUB_USERNAME }}
-          registry_password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-          sbom-predicate-type: ${{ env.SBOM_PREDICATE_TYPE }}
-          sbom-repository: docker.io/${{ vars.DOCKER_HUB_USERNAME }}/sbom
+          - name: backend
+            image_repo: app-stayhealthy-backend
+            image_digest: ${{ needs.build-push-backend.outputs.image_digest }}
```

actionlint (1.7.12)

[error] property "image_digest" is not defined in object type {image-digest: string}

(expression)